backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -24,6 +24,12 @@ You can schedule backups by creating a xref:../../../backup_and_restore/applicat
 
 include::modules/oadp-creating-backup-cr.adoc[leveloffset=+1]
 include::modules/oadp-backing-up-pvs-csi.adoc[leveloffset=+1]
-include::modules/oadp-backing-up-applications-restic.adoc[leveloffset=+1]
```

Comment:
There are several xrefs to the heading in this module. Please search your branch for "oadp-backing-up-applications-restic" and update the links.

Reply:
@kalexand-rh 
Done